examples/rating/app.R

```diff
@@ -0,0 +1,22 @@
+library(shiny)
+library(shiny.semantic)
+
+ui <- function() {
+  shinyUI(
```

Comment:
I think there's been folder confusion here. This should be (was) dropdown example ;)

Reply:
Sorry, fixed :)